script/django/setup.bat

```diff
@@ -0,0 +1,7 @@
+pushd django
+python3 -m venv .
+CALL .\Scripts\activate.bat
```

Comment:
Need to use `CALL` as this call `activate.bat` as a sub-process. If we don't the script ends right after this line without finishing the other commands